token/program2/src/instruction.rs

```diff
@@ -187,6 +192,23 @@ pub enum TokenInstruction {
     ///   2. `[]` The account's multisignature owner.
     ///   3. ..3+M '[signer]' M signer accounts.
     CloseAccount,
+    /// Freeze an Initialized account or unfreeze a Frozen account, using the Mint's
+    /// freeze_authority (if set).
+    /// Native accounts cannot be frozen
```

Comment:
I don't think this needs to be called out here.  The native mint will just not support freezing, so that documentation belongs with the native mint definition